.yarn/patches/@nx-js-npm-20.0.7-30719000fd.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/executors/release-publish/release-publish.impl.js b/src/executors/release-publish/release-publish.impl.js
+index 08cd58274d308bcb2ca644e8af5bd99a153c9e91..5ce12c3cb563345ffd19159fd108ff763626b930 100644
+--- a/src/executors/release-publish/release-publish.impl.js
```

Comment:
err are line 3 and 4 expected? they kind of look like random lines from a git merge conflict

Reply:
Yeah, its' how patches work